packages/wallets/src/features/accounts/modules/Poa/components/CommonMistakesExamples/CommonMistakesExamples.scss

```diff
@@ -30,6 +30,7 @@
                 margin-inline: auto;
             }
         }
+
```

Comment:
can we undo this aswell to reduce the file count

Reply:
Updated